yarn.config.cjs

```diff
@@ -165,38 +165,45 @@ module.exports = defineConfig({
           getRelativePath(workspace, 'scripts', 'since-latest-release.sh'),
         );
 
-        if (workspace.cwd !== 'packages/examples') {
+        if (
+          workspace.cwd !== 'packages/examples' &&
+          workspace.cwd !== 'packages/snaps-simulator'
```

Comment:
For some reason `--experimental-vm-modules` causes the `snaps-simulator` tests to break.